jekyll/_cci2/hello-world-macos.md

```diff
@@ -46,8 +46,9 @@ As a user getting to know the macOS execution environment, our ideal scenario is
 - Run tests using XCode on the macOS VM whenever we push code.
 - Create and upload the compiled application as an artifact after tests have run successfully.
 
-You can checkout the example application's repo on
-[GitHub](https://github.com/CircleCI-Public/circleci-demo-macos).
+You can checkout the example application's repo on [GitHub](https://github.com/CircleCI-Public/circleci-demo-macos).
```

Comment:
As it's a verb here, shouldn't _check out_ be two words?

Reply:
lol @catherinepope I almost asked about this. I'm not sure the original context behind the wording, but I took this to mean like "hey! check it out!" and then I second guessed myself if that was one word or two